asv/console.py

```diff
@@ -218,8 +218,8 @@ def _stream_formatter(self, record):
         continued = getattr(record, 'continued', False)
 
         if self._total:
-            progress_msg = '[{0:6.02f}%] '.format(
-                (float(self._count) / self._total) * 100.0)
+            percentage = (float(self._count) / self._total) * 100.0
+            progress_msg = f'[{percentage:6.02f}%] '
```

Comment:
The original code wasn't great. In Python you can actually do this:
```python
>>> value = .2
>>> f'{value:.2%}'
'20.00%'
```

Also, converting `self._count` to `float` only made sense in Python 2, in Python 3 it's not needed. So, this could be something like (make sure we're not changing the result):
```python
progress_msg = f'[{self._count / self._total:6.02%}] '
```

Reply:
Done! thanks for the suggestion :)